test/Microsoft.Framework.DependencyInjection.Tests/ActivatorUtilitiesTests.cs

```diff
@@ -79,12 +79,24 @@ public void TypeActivatorWorksWithStaticCtor(CreateInstanceFunc createFunc)
             Assert.NotNull(anotherClass);
         }
 
-        [Theory]
-        [MemberData(nameof(CreateInstanceFuncs))]
-        public void TypeActivatorWorksWithCtorWithOptionalArgs(CreateInstanceFunc createFunc)
```

Comment:
This test shouldn't have broke.

`CreateInstance` will allow for missing services if there is a default value for the parameter. `CreateFactory` used to work the same way in this regard. Now it doesn't.